tools/generators/migrate-converged-pkg/README.md

```diff
@@ -22,29 +22,29 @@ Workspace Generator for migrating converged packages to new DX (stage 1)
 ## Usage
 
 ```sh
-yarn nx workspace-generator migrate-converged-pkg ...
+yarn nx workspace-generate migrate-converged-pkg ...
```

Comment:
Is this intentional or just a bad merge?

Reply:
this change should be reverted